pkg/pxc/service.go

```diff
@@ -338,3 +360,10 @@ func NewServiceHAProxyReplicas(cr *api.PerconaXtraDBCluster) *corev1.Service {
 
 	return obj
 }
+
+func NewServiceHAProxyReplicasName(cr *api.PerconaXtraDBCluster) types.NamespacedName {
```

Comment:
Consider rename to HAProxyReplicasNamespacedName

Reply:
fbb29694